kind/images.json

```diff
@@ -1,4 +1,15 @@
 [
+
+  {
+    "k8s": "1.27",
+    "version": "1.27.1",
+    "sha": "kindest/node:v1.27.1@sha256:c1fea3acc2b38dd39ad2cb2ab0aa012720283e03364e314b715c395624476c74"
+  },
```

Comment:
There is already an entry for 1.27, why are you adding a new one ? IF there are 2 entries for `1.27`, can the script pickup the latest of `1.27` ?

Reply:
fixed